core/src/test/java/com/datastax/oss/driver/internal/core/session/throttling/ConcurrencyLimitingRequestThrottlerTest.java

```diff
@@ -236,7 +239,116 @@ public void should_reject_enqueued_when_closing() {
     throttler.register(request);
 
     // Then
-    assertThatStage(request.started)
+    assertThatStage(request.ended)
         .isFailed(error -> assertThat(error).isInstanceOf(RequestThrottlingException.class));
   }
+
+  @Test
+  public void should_run_throttle_callbacks_concurrently()
+      throws ExecutionException, InterruptedException, TimeoutException {
```

Comment:
small nit: `ExecutionException` and `TimeoutException` are no longer thrown by code in this method, so those can be removed from the signature